### ISSUE TITLE
quotes in <value> auto-escaped

### ISSUE THREAD

**Issue by TK009** [AaltoAsia/O-MI#19]
Problems
=======

1. O-MI node accepts values containing quotes without escaping, but automatically escapes it in read results. (Which is correct escape or not to escape?)
2. Read result is escaped twice

Testing write request
=================

```
<omiEnvelope xmlns="http://www.opengroup.org/xsd/omi/1.0/" version="1.0" ttl="0">
  <write msgformat="odf">
    <msg>
      <Objects xmlns="http://www.opengroup.org/xsd/odf/1.0/">
        <Object>
          <id>MyObject</id>
          <InfoItem name="quotesInValue">
            <value type="xs:string">"test"</value>
          </InfoItem>
          <InfoItem name="ampInValue">
            <value>test&amp;test</value>
          </InfoItem>
        </Object>
      </Objects>
    </msg>
  </write>
</omiEnvelope>
```

Read result
=========

```
<omiEnvelope ttl="10" version="1.0" xmlns="http://www.opengroup.org/xsd/omi/1.0/">
  <response>
    <result msgformat="odf">
      <return returnCode="200">
      </return>
      <msg>
        <Objects xmlns="http://www.opengroup.org/xsd/odf/1.0/">
          <Object>
            <id>MyObject</id>
            <InfoItem name="quotesInValue">
              <value unixTime="1541609326"
                dateTime="2018-11-07T18:48:46.359+02:00"
                >&quot;test&quot;</value>
            </InfoItem>
            <InfoItem name="ampInValue">
              <value unixTime="1541609715"
                dateTime="2018-11-07T18:55:15.222+02:00"
                >test&amp;test</value>
            </InfoItem>
          </Object>
        </Objects>
      </msg>
    </result>
  </response>
</omiEnvelope>
```

Expected read result (?)
==============

```
<omiEnvelope ttl="10" version="1.0" xmlns="http://www.opengroup.org/xsd/omi/1.0/">
  <response>
    <result msgformat="odf">
      <return returnCode="200">
      </return>
      <msg>
        <Objects xmlns="http://www.opengroup.org/xsd/odf/1.0/">
          <Object>
            <id>MyObject</id>
            <InfoItem name="quotesInValue">
              <value unixTime="1541609326"
                dateTime="2018-11-07T18:48:46.359+02:00"
                >"test"</value>
            </InfoItem>
            <InfoItem name="ampInValue">
              <value unixTime="1541609715"
                dateTime="2018-11-07T18:55:15.222+02:00"
                >test&amp;test</value>
            </InfoItem>
          </Object>
        </Objects>
      </msg>
    </result>
  </response>
</omiEnvelope>
```

**Comment by TK009** [comment]
Double escaping fixed for v1.0.8 release

**Comment by TK009** [comment]
dependency issue: https://github.com/scala/scala-xml/issues/57

**Comment by TK009** [comment]
Fixed in [version 2.0.0](https://github.com/AaltoAsia/O-MI/releases/tag/2.0.0)
